test/integration/tags/for.ts

```diff
@@ -46,7 +46,7 @@ describe('tags/for', function () {
   it('should output forloop', async function () {
     const src = '{%for i in (1..1)%}{{forloop}}{%endfor%}'
     const html = await liquid.parseAndRender(src, scope)
-    return expect(html).to.equal('{"i":0,"length":1,"name":"i-(1..1)"}')
+    return expect(html).to.equal('{"i":0,"length":1,"name":"i-(1..1)","parentloop":{}}')
```

Comment:
Should this contain `parentloop: {}`? It's `nil` when no parent exsit, I guess it should be `null` if it does need to appear here.

Reply:
Of course. I've added a `toJSON()` method to `NullDrop`.